### PR TITLE
@ashfurrow => Add support for grabbing show artworks

### PR DESCRIPTION
**Pull request by orta**
https://github.com/artsy/Emergence/commit/91392059d8090c94430a9e24fc653bd28a93dd33
- Added a simple Partner model
- Made the Artwork struct decodable 
- Added an API route for the shows artworks

### CODE REVIEW

**Comment by orta** [comment]
This is reviewable now :+1:

File: Emergence/Contexts/Presenting Locations/ShowLocationsViewController.swift [hunk 1]
```diff
@@ -19,10 +19,9 @@ class ShowLocationsViewController: UIPageViewController, UIPageViewControllerDat
         let network = appVC.context.network
         let coords = location.coordinates()
         let showInfo = ArtsyAPI.RunningShowsNearLocation(amount: 2, lat: coords.lat, long: coords.long)
-        network.request(showInfo).mapSuccessfulHTTPToObjectArray(Show).subscribe(next: { decodables in
-            print("decodables: \(decodables)")
-                self.shows = decodables
+        network.request(showInfo).mapSuccessfulHTTPToObjectArray(Show).subscribe(next: { shows in
```
Comment (ashfurrow) [inline Emergence/Contexts/Presenting Locations/ShowLocationsViewController.swift:22]: Possible reference cycle with `self` here, if the VC is deallocated before the request completes.

File: Emergence/Contexts/Presenting Locations/ShowLocationsViewController.swift [hunk 2]
```diff
@@ -52,19 +51,15 @@ class ShowLocationsViewController: UIPageViewController, UIPageViewControllerDat
...
-            return nil
-        }
-        guard let sb = self.storyboard else {
-            return nil
-        }
+        guard -1...shows.count ~= index else { return nil }
+        guard let sb = self.storyboard else { return nil }
 
-        guard let controller = sb.instantiateViewControllerWithIdentifier("show") as? ShowViewController else {
+        guard let controller = sb.instantiateViewControllerWithIdentifier("show") as? ShowViewController else  {
```
Comment (ashfurrow) [inline Emergence/Contexts/Presenting Locations/ShowLocationsViewController.swift:57]: sad spacing panda :panda_face:

File: Emergence/Models/Electric Objects/Artwork.swift [hunk 3]
```diff
@@ -1,10 +1,32 @@
...
     let images: [Imageable]
 }
+
+extension Artwork: Decodable {
+    init?(json: JSON) {
+
+        guard
+            let idValue: String = "name" <~~ json,
+            let titleValue: String = "name" <~~ json,
+            let mediumValue: String = "medium" <~~ json
```
Comment (ashfurrow) [inline Emergence/Models/Electric Objects/Artwork.swift:19]: Really liking this pattern.

**Comment by ashfurrow** [comment]
Looks good! :100:

**Pull request merged**